gnomad_hail/utils/generic.py

```diff
@@ -238,12 +246,14 @@ def filter_low_conf_regions(mt: Union[hl.MatrixTable, hl.Table], filter_lcr: boo
     """
     Filters low-confidence regions
 
-    :param MatrixTable or Table mt: MatrixTable or Table to filter
+    :param mt: MatrixTable or Table to filter
+    :param mt: MatrixTable or Table
```

Comment:
should be `type`

Reply:
Fixed in bfa3b1a55893f55ae8c532f4c04890b68e2a005e